teamcity: failed tests on master: acceptance/TestEventLog, test/TestBackupRestoreControlJob/cancel, test/TestBackupRestoreControlJob, test/TestBackupRestoreControlJob/pause

The following tests appear to have failed:

[#323319](https://teamcity.cockroachdb.com/viewLog.html?buildId=323319):

```
--- FAIL: acceptance/TestEventLog (5.660s)
test_log_scope.go:78: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestEventLog764266838
test_log_scope.go:61: use -show-logs to present logs inline
panic: Error response from daemon: Get https://registry-1.docker.io/v2/library/ubuntu/manifests/xenial-20170214: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fubuntu%3Apull&service=registry.docker.io: dial tcp 52.206.159.189:443: connect: network is unreachable [recovered]
panic: Error response from daemon: Get https://registry-1.docker.io/v2/library/ubuntu/manifests/xenial-20170214: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fubuntu%3Apull&service=registry.docker.io: dial tcp 52.206.159.189:443: connect: network is unreachable
/usr/local/go/src/testing/testing.go:622 +0x29d
/usr/local/go/src/runtime/panic.go:489 +0x2cf
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/cluster/localcluster.go:273 +0xb1
/usr/local/go/src/runtime/panic.go:489 +0x2cf
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/cluster/docker.go:153 +0x4a
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/cluster/localcluster.go:571 +0x2a3
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/cluster/localcluster.go:695 +0x542
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/util.go:414 +0x191
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/util.go:378 +0x80
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/util.go:382 +0x1c6
/go/src/github.com/cockroachdb/cockroach/pkg/acceptance/event_log_test.go:39 +0xac
/usr/local/go/src/testing/testing.go:657 +0x96
/usr/local/go/src/testing/testing.go:697 +0x2ca
------- Stdout: -------
xenial-20170214: Pulling from library/ubuntu
Digest: sha256:dd7808d8792c9841d0b460122f1acf0a2dd1f56404f8d1e56298048885e45535
Status: Image is up to date for ubuntu:xenial-20170214

ERROR: a panic has occurred!
Details cannot be printed yet because we are still unwinding.
Hopefully the test harness prints the panic below, otherwise check the test logs.

test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestEventLog764266838
panic: Error response from daemon: Get https://registry-1.docker.io/v2/library/ubuntu/manifests/xenial-20170214: Get https://auth.docker.io/token?scope=repository%3Alibrary%2Fubuntu%3Apull&service=registry.docker.io: dial tcp 52.206.159.189:443: connect: network is unreachable [recovered]

goroutine 3533 [running]:
testing.tRunner.func1(0xc4209fa9c0)
panic(0x18816a0, 0xc42016de40)
github.com/cockroachdb/cockroach/pkg/acceptance/cluster.(*LocalCluster).stopOnPanic(0xc4207387e0, 0x7ff7a6fba858, 0xc420018de8)
panic(0x18816a0, 0xc42016de40)
github.com/cockroachdb/cockroach/pkg/acceptance/cluster.maybePanic(0x26922a0, 0xc42016de40)
github.com/cockroachdb/cockroach/pkg/acceptance/cluster.(*LocalCluster).RunInitCommand(0xc4207387e0, 0x7ff7a6fba858, 0xc420018de8, 0x0)
github.com/cockroachdb/cockroach/pkg/acceptance/cluster.(*LocalCluster).Start(0xc4207387e0, 0x7ff7a6fba858, 0xc420018de8)
github.com/cockroachdb/cockroach/pkg/acceptance.StartCluster(0x7ff7a6fba858, 0xc420018de8, 0xc4209fa9c0, 0xc4201ec8f0, 0x9, 0xc4202d6de0, 0x1, 0x1, 0x12a05f200, 0x0, ...)
github.com/cockroachdb/cockroach/pkg/acceptance.runTestOnConfigs.func1(0x7ff7a6fba858, 0xc420018de8, 0xc4209fa9c0, 0xc420422bc0, 0x1b2f918)
github.com/cockroachdb/cockroach/pkg/acceptance.runTestOnConfigs(0xc4209fa9c0, 0x1b2f918, 0x0, 0x0, 0x0)
github.com/cockroachdb/cockroach/pkg/acceptance.TestEventLog(0xc4209fa9c0)
testing.tRunner(0xc4209fa9c0, 0x1b2f828)
created by testing.(*T).Run
--- FAIL: test/TestBackupRestoreControlJob/cancel (0.000s)
Test ended in panic.

------- Stdout: -------
I170814 15:53:17.301141 57982 sql/event_log.go:99  [client=127.0.0.1:39418,user=root,n1] Event: "create_database", target: 54, info: {DatabaseName:cancel Statement:CREATE DATABASE cancel User:root}
I170814 15:53:17.301904 54011 storage/replica_command.go:2704  [split,n1,s1,r27/1:/Table/5{2-3/1}] initiating a split of this range at key /Table/53 [r38]
I170814 15:53:17.310030 54011 storage/replica_command.go:2704  [split,n1,s1,r37/1:/{Table/53/1/9���-Max}] initiating a split of this range at key /Table/54 [r39]
I170814 15:53:17.634680 54015 storage/replica_raftstorage.go:509  [raftsnapshot,n1,s1,r20/1:/Table/51/1/{300-400}] generated Raft snapshot 69a3cd3a at index 15
I170814 15:53:17.635580 54015 storage/store.go:3432  [raftsnapshot,n1,s1,r20/1:/Table/51/1/{300-400}] streamed snapshot to (n2,s2):3: kv pairs: 110, log entries: 5, rate-limit: 8.0 MiB/sec, 1ms
I170814 15:53:17.635768 61074 storage/replica_raftstorage.go:718  [n2,s2,r20/3:{-}] applying Raft snapshot at index 15 (id=69a3cd3a, encoded size=27824, 1 rocksdb batches, 5 log entries)
I170814 15:53:17.636287 61074 storage/replica_raftstorage.go:726  [n2,s2,r20/3:/Table/51/1/{300-400}] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170814 15:53:17.637082 54015 storage/replica_raftstorage.go:509  [raftsnapshot,n1,s1,r21/1:/Table/51/1/{400-500}] generated Raft snapshot ebc4ea01 at index 18
I170814 15:53:17.637868 54015 storage/store.go:3432  [raftsnapshot,n1,s1,r21/1:/Table/51/1/{400-500}] streamed snapshot to (n2,s2):3: kv pairs: 110, log entries: 8, rate-limit: 8.0 MiB/sec, 1ms
I170814 15:53:17.638163 61063 storage/replica_raftstorage.go:718  [n2,s2,r21/3:{-}] applying Raft snapshot at index 18 (id=ebc4ea01, encoded size=29601, 1 rocksdb batches, 8 log entries)
I170814 15:53:17.638667 61063 storage/replica_raftstorage.go:726  [n2,s2,r21/3:/Table/51/1/{400-500}] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170814 15:53:17.944406 55196 storage/replica_proposal.go:453  [n3,s3,r7/2:/Table/1{2-3}] new range lease repl=(n3,s3):2 start=1502725997.943375416,0 epo=1 pro=1502725997.943378216,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:18.234595 54015 storage/replica_raftstorage.go:509  [raftsnapshot,n1,s1,r36/1:/Table/53/1/{800-900}] generated Raft snapshot 20bd5d98 at index 17
I170814 15:53:18.235540 54015 storage/store.go:3432  [raftsnapshot,n1,s1,r36/1:/Table/53/1/{800-900}] streamed snapshot to (n2,s2):3: kv pairs: 110, log entries: 7, rate-limit: 8.0 MiB/sec, 1ms
I170814 15:53:18.235696 61105 storage/replica_raftstorage.go:718  [n2,s2,r36/3:/{Table/53/1/8���-Max}] applying Raft snapshot at index 17 (id=20bd5d98, encoded size=47596, 1 rocksdb batches, 7 log entries)
I170814 15:53:18.236297 61105 storage/replica_raftstorage.go:726  [n2,s2,r36/3:/Table/53/1/{800-900}] applied Raft snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170814 15:53:18.236787 54015 storage/replica_raftstorage.go:509  [raftsnapshot,n1,s1,r37/1:/Table/5{3/1/900-4}] generated Raft snapshot 37305e09 at index 17
I170814 15:53:18.237760 54015 storage/store.go:3432  [raftsnapshot,n1,s1,r37/1:/Table/5{3/1/900-4}] streamed snapshot to (n2,s2):3: kv pairs: 110, log entries: 7, rate-limit: 8.0 MiB/sec, 1ms
I170814 15:53:18.237944 61298 storage/replica_raftstorage.go:718  [n2,s2,r37/3:{-}] applying Raft snapshot at index 17 (id=37305e09, encoded size=65894, 1 rocksdb batches, 7 log entries)
I170814 15:53:18.238699 61298 storage/replica_raftstorage.go:726  [n2,s2,r37/3:/Table/5{3/1/900-4}] applied Raft snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170814 15:53:18.947465 54976 storage/replica_proposal.go:453  [n2,s2,r25/3:/Table/51/1/{800-900}] new range lease repl=(n2,s2):3 start=1502725998.945769578,0 epo=1 pro=1502725998.945772178,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:19.949546 55187 storage/replica_proposal.go:453  [n3,s3,r26/2:/Table/5{1/1/900-2}] new range lease repl=(n3,s3):2 start=1502725999.947720521,0 epo=1 pro=1502725999.947723021,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:20.267533 55256 storage/replica_raftstorage.go:509  [raftsnapshot,n3,s3,r39/2:/{Table/54-Max}] generated Raft snapshot 3c42609a at index 11
I170814 15:53:20.268093 55256 storage/store.go:3432  [raftsnapshot,n3,s3,r39/2:/{Table/54-Max}] streamed snapshot to (n2,s2):3: kv pairs: 9, log entries: 1, rate-limit: 8.0 MiB/sec, 0ms
I170814 15:53:20.268236 62002 storage/replica_raftstorage.go:718  [n2,s2,r39/3:{-}] applying Raft snapshot at index 11 (id=3c42609a, encoded size=559, 1 rocksdb batches, 1 log entries)
I170814 15:53:20.268455 62002 storage/replica_raftstorage.go:726  [n2,s2,r39/3:/{Table/54-Max}] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170814 15:53:20.643064 54232 storage/replica_proposal.go:453  [n1,s1,r5/1:/Table/{SystemCon���-11}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726000.642315113,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:20.647222 54238 storage/replica_proposal.go:453  [n1,s1,r10/1:/Table/1{5-6}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726000.646140599,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:21.350627 54250 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r23/1:/Table/51/1/{600-700}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726001.349856336,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:21.551023 54237 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r30/1:/Table/53/1/{200-300}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726001.550215584,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:21.690234 55239 storage/replica_proposal.go:453  [replica consistency checker,n3,s3,r39/2:/{Table/54-Max}] new range lease repl=(n3,s3):2 start=1502726001.130768204,1 epo=1 pro=1502726001.688778825,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:21.890442 55236 storage/replica_proposal.go:453  [replica consistency checker,n3,s3,r33/2:/Table/53/1/{500-600}] new range lease repl=(n3,s3):2 start=1502726001.130768204,1 epo=1 pro=1502726001.889103672,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:21.951419 54272 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r27/1:/Table/5{2-3}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726001.950765972,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:22.553945 54266 storage/replica_proposal.go:453  [n1,s1,r38/1:/Table/53{-/1}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726002.552923270,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:22.752983 54256 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r18/1:/Table/51/1/{100-200}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726002.752276869,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:23.153600 54276 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r19/1:/Table/51/1/{200-300}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726003.152858560,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:23.693561 55222 storage/replica_proposal.go:453  [replica consistency checker,n3,s3,r22/2:/Table/51/1/{500-600}] new range lease repl=(n3,s3):2 start=1502726001.130768204,1 epo=1 pro=1502726003.692202003,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:24.155040 54240 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r20/1:/Table/51/1/{300-400}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726004.154247085,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:24.494720 54640 storage/replica_proposal.go:453  [n2,s2,r8/2:/Table/1{3-4}] new range lease repl=(n2,s2):2 start=1502726004.493658501,0 epo=1 pro=1502726004.493661201,0 following repl=(n3,s3):3 start=1502725995.937580650,0 epo=1 pro=1502725995.937582850,0
I170814 15:53:25.156642 54254 storage/replica_proposal.go:453  [replica consistency checker,n1,s1,r35/1:/Table/53/1/{700-800}] new range lease repl=(n1,s1):1 start=0.000000000,0 epo=1 pro=1502726005.155838823,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1502726001.130768204,0 pro=1502725992.130785505,0
I170814 15:53:36.144944 54304 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:36714->127.0.0.1:39338: use of closed network connection
I170814 15:53:36.144982 54419 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:53:36.144994 54419 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:53:36.250130 54865 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:53:36.250153 54865 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:53:36.250159 55014 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:32789->127.0.0.1:46352: use of closed network connection
I170814 15:53:36.277054 55179 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:40667->127.0.0.1:50785: use of closed network connection
I170814 15:53:36.277087 55182 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:53:36.277097 55182 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:53:42.676824 54155 gossip/gossip.go:1218  [n1] node has connected to cluster via gossip
I170814 15:53:42.677016 54155 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170814 15:53:43.636734 54834 gossip/gossip.go:1218  [n2] node has connected to cluster via gossip
I170814 15:53:43.636851 54834 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170814 15:53:45.121941 55088 gossip/gossip.go:1218  [n3] node has connected to cluster via gossip
I170814 15:53:45.122083 55088 storage/stores.go:312  [n3] wrote 2 node addresses to persistent storage
I170814 15:54:12.131517 54309 gossip/gossip.go:446  [n1] gossip status (ok, 3 nodes)
gossip client (0/3 cur/max conns)
gossip server (2/3 cur/max conns, infos 629/119 sent/received, bytes 205489B/34585B sent/received)
  2: 127.0.0.1:32789 (1m0s)
  3: 127.0.0.1:40667 (1m0s)
I170814 15:54:12.233511 54817 gossip/gossip.go:446  [n2] gossip status (ok, 3 nodes)
gossip client (1/3 cur/max conns)
  1: 127.0.0.1:36714 (1m0s: infos 53/341 sent/received, bytes 16047B/101662B sent/received)
gossip server (0/3 cur/max conns, infos 53/341 sent/received, bytes 16047B/101662B sent/received)
I170814 15:54:12.259907 55139 gossip/gossip.go:446  [n3] gossip status (ok, 3 nodes)
gossip client (1/3 cur/max conns)
  1: 127.0.0.1:36714 (1m0s: infos 67/330 sent/received, bytes 18843B/109501B sent/received)
gossip server (0/3 cur/max conns, infos 67/330 sent/received, bytes 18843B/109501B sent/received)
I170814 15:54:15.155761 66871 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:36714->127.0.0.1:39672: use of closed network connection
I170814 15:54:15.155796 66903 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:54:15.155805 66903 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:54:15.261049 66567 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:54:15.261092 66564 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:32789->127.0.0.1:46685: use of closed network connection
I170814 15:54:15.261121 66567 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:54:15.287505 66938 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:54:15.287531 66937 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:40667->127.0.0.1:51117: use of closed network connection
I170814 15:54:15.287551 66938 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:55:12.131816 54309 gossip/gossip.go:446  [n1] gossip status (ok, 3 nodes)
gossip client (0/3 cur/max conns)
gossip server (2/3 cur/max conns, infos 980/125 sent/received, bytes 245758B/36045B sent/received)
  2: 127.0.0.1:32789 (2m0s)
  3: 127.0.0.1:40667 (2m0s)
I170814 15:55:12.233377 54817 gossip/gossip.go:446  [n2] gossip status (ok, 3 nodes)
gossip client (1/3 cur/max conns)
  1: 127.0.0.1:36714 (2m0s: infos 56/550 sent/received, bytes 16734B/126983B sent/received)
gossip server (0/3 cur/max conns, infos 56/550 sent/received, bytes 16734B/126983B sent/received)
I170814 15:55:12.259979 55139 gossip/gossip.go:446  [n3] gossip status (ok, 3 nodes)
gossip client (1/3 cur/max conns)
  1: 127.0.0.1:36714 (2m0s: infos 69/472 sent/received, bytes 19311B/123849B sent/received)
gossip server (0/3 cur/max conns, infos 69/472 sent/received, bytes 19311B/123849B sent/received)
I170814 15:55:24.166516 78395 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:36714->127.0.0.1:40105: use of closed network connection
I170814 15:55:24.166624 78396 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:55:24.166642 78396 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:55:24.271329 78337 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:32789->127.0.0.1:47119: use of closed network connection
I170814 15:55:24.271400 78057 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:55:24.271419 78057 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
I170814 15:55:24.298132 78552 vendor/google.golang.org/grpc/grpclog/grpclog.go:75  transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:40667->127.0.0.1:51551: use of closed network connection
I170814 15:55:24.298167 78555 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  Client received GoAway with http2.ErrCodeEnhanceYourCalm.
I170814 15:55:24.298181 78555 vendor/google.golang.org/grpc/grpclog/grpclog.go:60  transport: http2Client.notifyError got notified that the client transport was broken EOF.
panic: test timed out after 4m0s

goroutine 106780 [running]:
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1023 +0xf9
created by time.goFunc
	/usr/local/go/src/time/sleep.go:170 +0x44

goroutine 1 [chan receive, 2 minutes]:
testing.(*T).Run(0xc4204d4410, 0x1c06631, 0x1b, 0x1cb8738, 0xc4205ffc01)
	/usr/local/go/src/testing/testing.go:698 +0x2f4
testing.runTests.func1(0xc4204d4410)
	/usr/local/go/src/testing/testing.go:882 +0x67
testing.tRunner(0xc4204d4410, 0xc4205ffda0)
	/usr/local/go/src/testing/testing.go:657 +0x96
testing.runTests(0xc420370f60, 0x28ec7e0, 0x26, 0x26, 0x2db2b60)
	/usr/local/go/src/testing/testing.go:888 +0x2c1
testing.(*M).Run(0xc420db3f20, 0x2df3600)
	/usr/local/go/src/testing/testing.go:822 +0xfc
github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.TestMain(0xc4205fff20)
	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/main_test.go:31 +0xfc
main.main()
	github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/_test/_testmain.go:130 +0xf7

goroutine 17 [syscall, 4 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2197 +0x1

goroutine 6 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x2db4620)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:988 +0x77
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:575 +0xd9

goroutine 15 [syscall, 4 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x104
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x41

goroutine 34 [select, 2 minutes, locked to thread]:
runtime.gopark(0x1cbbed0, 0x0, 0x1be7e7e, 0x6, 0x18, 0x2)
	/usr/local/go/src/runtime/proc.go:271 +0x13a
runtime.selectgoImpl(0xc4209d4f50, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:423 +0x1364
runtime.selectgo(0xc4209d4f50)
	/usr/local/go/src/runtime/select.go:238 +0x1c
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal_unix.go:434 +0x2dd
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2197 +0x1

goroutine 53914 [chan receive, 2 minutes]:
testing.(*T).Run(0xc420627450, 0x1be7500, 0x6, 0xc424293ec0, 0x1cb8801)
	/usr/local/go/src/testing/testing.go:698 +0x2f4
github.com/cockroachdb/cockroach/pkg/ccl/sqlccl_test.TestBackupRestoreControlJob(0xc420627450)
	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup_test.go:913 +0x3fe
testing.tRunner(0xc420627450, 0x1cb8738)
	/usr/local/go/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:697 +0x2ca

goroutine 55294 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).RaftMessageBatch(0xc420ae63c0, 0x291b3e0, 0xc424abce70, 0x2901100, 0xc420ae63c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:356 +0x23d
github.com/cockroachdb/cockroach/pkg/storage._MultiRaft_RaftMessageBatch_Handler(0x1bb6aa0, 0xc420ae63c0, 0x29182c0, 0xc421470bd0, 0xc424ecba40, 0x2910780)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft.pb.go:378 +0xb7
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).processStreamingRPC(0xc421021e00, 0x29196a0, 0xc423f5c6e0, 0xc420ab0c00, 0xc4218d4510, 0x28e1ec0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:891 +0xa4c
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleStream(0xc421021e00, 0x29196a0, 0xc423f5c6e0, 0xc420ab0c00, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:985 +0x153f
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc422d72590, 0xc421021e00, 0x29196a0, 0xc423f5c6e0, 0xc420ab0c00)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:551 +0xa9
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:552 +0xa1

goroutine 54932 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a5b)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc420400270)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420400270)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9e840, 0xc423f98cf0, 0xc424f9e830)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55825 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).keepalive(0xc4227bda00)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1276 +0x4ed
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:294 +0xefe

goroutine 54986 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a6a)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc4204018c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc4204018c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9ef70, 0xc423f98cf0, 0xc424f9ef60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55231 [semacquire]:
sync.runtime_notifyListWait(0xc420d5f090, 0xc400007b76)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420d5f080)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4227ee5a0, 0x7fb9281090a8, 0xc42391f0b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc42391f0b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d08870, 0xc420f2e240, 0xc421d08860)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54973 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a85)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc420401530)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420401530)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9edc0, 0xc423f98cf0, 0xc424f9edb0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55191 [semacquire]:
sync.runtime_notifyListWait(0xc420d5f090, 0xc400007b8d)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420d5f080)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4227ee5a0, 0x7fb9281090a8, 0xc42391e690)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc42391e690)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d08360, 0xc420f2e240, 0xc421d08340)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54901 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*SchemaChangeManager).Start.func1(0x7fb9281090a8, 0xc4219f4db0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:973 +0x163a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203ed8c0, 0xc423f98cf0, 0xc4202dd1c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54335 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc420c19020)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420398e70, 0xc42458a1b0, 0xc420398e60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54900 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc4219f4d80)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203ed8a0, 0xc423f98cf0, 0xc4203ed890)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55138 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/server.(*Node).startComputePeriodicMetrics.func1(0x7fb9281090a8, 0xc4227b4330)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:650 +0x223
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420368630, 0xc420f2e240, 0xc422814460)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54592 [chan receive, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/gossip.(*server).start.func3(0x7fb9281090a8, 0xc424c34060)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:384 +0x66
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027e450, 0xc423f98cf0, 0xc422566400)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54961 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a5d)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc420401140)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420401140)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9ec40, 0xc423f98cf0, 0xc424f9ec30)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55243 [semacquire, 2 minutes]:
sync.runtime_Semacquire(0xc4227ee624)
	/usr/local/go/src/runtime/sema.go:47 +0x34
sync.(*WaitGroup).Wait(0xc4227ee618)
	/usr/local/go/src/sync/waitgroup.go:131 +0x7a
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait(0xc4227ee5a0, 0x7fb9281090a8, 0xc42391f320)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:173 +0x31
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait-fm(0x7fb9281090a8, 0xc42391f320)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3559 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d08a00, 0xc420f2e240, 0xc421d089f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54871 [semacquire, 2 minutes]:
sync.runtime_SemacquireMutex(0xc42027e224)
	/usr/local/go/src/runtime/sema.go:62 +0x34
sync.(*Mutex).Lock(0xc42027e220)
	/usr/local/go/src/sync/mutex.go:87 +0x9d
sync.(*Once).Do(0xc42027e220, 0xc423d32f60)
	/usr/local/go/src/sync/once.go:40 +0x44
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func11(0x7fb9281090a8, 0xc422a1cf30)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:881 +0x64
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027fe40, 0xc423f98cf0, 0xc4219ec960)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54931 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a88)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc420400a50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420400a50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9e820, 0xc423f98cf0, 0xc424f9e810)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54769 [semacquire, 2 minutes]:
sync.runtime_notifyListWait(0xc4204d1b30, 0xc400000000)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc4204d1b20)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).syncLoop(0xc4204d1a40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:466 +0xd7
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:453 +0x4c7

goroutine 54939 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a60)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc420400c90)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420400c90)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9e920, 0xc423f98cf0, 0xc424f9e910)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54864 [select]:
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).runHeartbeat(0xc420637040, 0xc424e92c40, 0xc4202705f0, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:412 +0x7ef
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).GRPCDial.func1.2.1(0x7fb9281090a8, 0xc424057dd0)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:346 +0x6b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42534c720, 0xc420f2e240, 0xc424057d10)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 56211 [IO wait]:
net.runtime_pollWait(0x7fb925182e50, 0x72, 0x1f)
	/usr/local/go/src/runtime/netpoll.go:164 +0x59
net.(*pollDesc).wait(0xc42009d5d8, 0x72, 0x28fefc0, 0x28f3e30)
	/usr/local/go/src/net/fd_poll_runtime.go:75 +0x38
net.(*pollDesc).waitRead(0xc42009d5d8, 0xc423554000, 0x8000)
	/usr/local/go/src/net/fd_poll_runtime.go:80 +0x34
net.(*netFD).Read(0xc42009d570, 0xc423554000, 0x8000, 0x8000, 0x0, 0x28fefc0, 0x28f3e30)
	/usr/local/go/src/net/fd_unix.go:250 +0x1b7
net.(*conn).Read(0xc420de63c0, 0xc423554000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:181 +0x70
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*MuxConn).Read(0xc4225ce000, 0xc423554000, 0x8000, 0x8000, 0x1b0d660, 0x2df3600, 0x7fb92513d000)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:218 +0x113
crypto/tls.(*block).readFromUntil(0xc42356d3e0, 0x7fb92513d000, 0xc4225ce000, 0x5, 0xc4225ce000, 0xc422818000)
	/usr/local/go/src/crypto/tls/conn.go:488 +0x98
crypto/tls.(*Conn).readRecord(0xc4214f4700, 0x1cbbf17, 0xc4214f4820, 0xed123bf07)
	/usr/local/go/src/crypto/tls/conn.go:590 +0xc4
crypto/tls.(*Conn).Read(0xc4214f4700, 0xc420608000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1134 +0x11d
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*readTimeoutConn).Read(0xc42023b6c0, 0xc420608000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:162 +0x158
bufio.(*Reader).fill(0xc42311a4e0)
	/usr/local/go/src/bufio/bufio.go:97 +0x117
bufio.(*Reader).ReadByte(0xc42311a4e0, 0x6, 0x0, 0x0)
	/usr/local/go/src/bufio/bufio.go:239 +0x5b
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*readBuffer).readTypedMsg(0xc4236b7228, 0x2904c40, 0xc42311a4e0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/encoding.go:92 +0x35
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*v3Conn).serve(0xc4236b7200, 0x7fb9281090a8, 0xc424794f90, 0xc4203bb7e0, 0x5400, 0xc420ad8f68, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/v3.go:473 +0x44d
github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*Server).ServeConn(0xc420ad8e00, 0x7fb9281090e8, 0xc424570ac0, 0x291a240, 0xc4214f4700, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/server.go:421 +0x96c
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func9.1(0x291b500, 0xc4225ce000)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:733 +0x149
github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith.func1(0xc42458a1b0, 0x7fb9281090a8, 0xc424e9a060, 0xc420e5a128, 0x291b500, 0xc4225ce000, 0xc421e58030)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:140 +0xa9
created by github.com/cockroachdb/cockroach/pkg/util/netutil.(*Server).ServeWith
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:142 +0x267

goroutine 54315 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc420c19ef0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203989e0, 0xc42458a1b0, 0xc4203989c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55360 [IO wait]:
net.runtime_pollWait(0x7fb928109f88, 0x72, 0x18)
	/usr/local/go/src/runtime/netpoll.go:164 +0x59
net.(*pollDesc).wait(0xc420722928, 0x72, 0x28fefc0, 0x28f3e30)
	/usr/local/go/src/net/fd_poll_runtime.go:75 +0x38
net.(*pollDesc).waitRead(0xc420722928, 0xc4225d6000, 0x800)
	/usr/local/go/src/net/fd_poll_runtime.go:80 +0x34
net.(*netFD).Read(0xc4207228c0, 0xc4225d6000, 0x800, 0x800, 0x0, 0x28fefc0, 0x28f3e30)
	/usr/local/go/src/net/fd_unix.go:250 +0x1b7
net.(*conn).Read(0xc42000e3c0, 0xc4225d6000, 0x800, 0x800, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:181 +0x70
crypto/tls.(*block).readFromUntil(0xc421e5e4b0, 0x7fb92810f188, 0xc42000e3c0, 0x5, 0xc42000e3c0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:488 +0x98
crypto/tls.(*Conn).readRecord(0xc421236000, 0x1cbbf17, 0xc421236120, 0xc423727ad4)
	/usr/local/go/src/crypto/tls/conn.go:590 +0xc4
crypto/tls.(*Conn).Read(0xc421236000, 0xc421148000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1134 +0x11d
bufio.(*Reader).Read(0xc420d8b620, 0xc4221f9998, 0x9, 0x9, 0xc422a53d60, 0x6c0e2f, 0x1cbbff0)
	/usr/local/go/src/bufio/bufio.go:213 +0x312
io.ReadAtLeast(0x28f5b40, 0xc420d8b620, 0xc4221f9998, 0x9, 0x9, 0x9, 0x6, 0xc42001b968, 0xc42001b900)
	/usr/local/go/src/io/io.go:307 +0xa9
io.ReadFull(0x28f5b40, 0xc420d8b620, 0xc4221f9998, 0x9, 0x9, 0x0, 0x0, 0xc422a53e30)
	/usr/local/go/src/io/io.go:325 +0x58
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc4221f9998, 0x9, 0x9, 0x28f5b40, 0xc420d8b620, 0x0, 0xc400000000, 0xc422a53e75, 0xc4215c7890)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc4221f9960, 0x2901a40, 0xc4215c7890, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0xa4
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).readFrame(0xc4212152f0, 0xc4215c7890, 0xc4215c7890, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:592 +0x2f
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).reader(0xc420f4b860)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1154 +0xcd
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:262 +0xcf5

goroutine 55361 [select, 2 minutes]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).controller(0xc420f4b860)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1227 +0x6c7
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:292 +0xe3d

goroutine 54234 [semacquire]:
sync.runtime_notifyListWait(0xc4250ec0d0, 0xc400008281)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc4250ec0c0)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42458b3b0, 0x7fb9281090a8, 0xc424e9a8a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc424e9a8a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027e160, 0xc42458a1b0, 0xc42027e150)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55200 [semacquire]:
sync.runtime_notifyListWait(0xc420d5f090, 0xc400007ba5)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420d5f080)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4227ee5a0, 0x7fb9281090a8, 0xc42391e960)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc42391e960)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d08480, 0xc420f2e240, 0xc421d08470)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54943 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a76)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc425090120, 0x7fb9281090a8, 0xc420400d80)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420400d80)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9e9a0, 0xc423f98cf0, 0xc424f9e990)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54253 [semacquire]:
sync.runtime_notifyListWait(0xc4250ec0d0, 0xc400008297)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc4250ec0c0)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42458b3b0, 0x7fb9281090a8, 0xc423214480)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc423214480)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027e480, 0xc42458a1b0, 0xc42027e450)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54873 [select]:
github.com/cockroachdb/cockroach/pkg/sql/jobs.(*Registry).Start.func1(0x7fb9281090a8, 0xc424794000)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/jobs/registry.go:138 +0x18c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027fed0, 0xc423f98cf0, 0xc421f04930)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54269 [semacquire]:
sync.runtime_notifyListWait(0xc4250ec0d0, 0xc40000827c)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc4250ec0c0)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42458b3b0, 0x7fb9281090a8, 0xc424e9a900)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc424e9a900)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027e910, 0xc42458a1b0, 0xc42027e8f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54317 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc4201d0e70)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420398a50, 0xc42458a1b0, 0xc420398a20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54238 [semacquire]:
sync.runtime_notifyListWait(0xc4250ec0d0, 0xc40000829a)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc4250ec0c0)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42458b3b0, 0x7fb9281090a8, 0xc420cdbb30)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc420cdbb30)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42027e200, 0xc42458a1b0, 0xc42027e1f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55137 [semacquire]:
sync.runtime_notifyListWait(0xc420d5f090, 0xc400007b75)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420d5f080)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4227ee5a0, 0x7fb9281090a8, 0xc42391e570)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc42391e570)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d08270, 0xc420f2e240, 0xc421d08260)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54634 [chan receive, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/util/netutil.MakeServer.func2(0x7fb9281090a8, 0xc4233e81b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:100 +0x70
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424f9e030, 0xc423f98cf0, 0xc423332040)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55254 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x7fb9281090a8, 0xc42391f590)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:504 +0x42f
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d08b80, 0xc420f2e240, 0xc42203a280)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 98928 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).keepalive(0xc42227e820)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1276 +0x4ed
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:294 +0xefe

goroutine 54322 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc420c19050)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420398b00, 0xc42458a1b0, 0xc420398af0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54906 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc4219f5d70)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203ed9e0, 0xc423f98cf0, 0xc4203ed9d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55097 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).keepalive(0xc4224dc680)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1276 +0x4ed
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:294 +0xefe

goroutine 55188 [semacquire]:
sync.runtime_notifyListWait(0xc420d5f090, 0xc400007bb3)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420d5f080)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4227ee5a0, 0x7fb9281090a8, 0xc42391e600)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x7fb9281090a8, 0xc42391e600)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc421d082d0, 0xc420f2e240, 0xc421d082c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54909 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc4219f5e00)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203eda80, 0xc423f98cf0, 0xc4203eda50)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 55172 [select, 2 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1(0x7fb9281090a8, 0xc423b823c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:85 +0x181
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420368e40, 0xc420f2e240, 0xc420368e30)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf7
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:196 +0xad

goroutine 54975 [semacquire]:
sync.runtime_notifyListWait(0xc42545e990, 0xc400007a73)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc42545e980)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroa
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/jobs/registry.go
+++ b/pkg/sql/jobs/registry.go
@@ -15,8 +15,10 @@
 package jobs
 
 import (
+	"fmt"
 	"time"
 
+	"github.com/pkg/errors"
 	"golang.org/x/net/context"
 
 	"github.com/cockroachdb/cockroach/pkg/base"
@@ -31,7 +33,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/stop"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/cockroach/pkg/util/uuid"
-	"github.com/pkg/errors"
 )
 
 // nodeLiveness is the subset of storage.NodeLiveness's interface needed
@@ -291,7 +292,25 @@ func (r *Registry) maybeAdoptJob(ctx context.Context, nl nodeLiveness) error {
 		go func() {
 			log.Infof(ctx, "job %d: resuming", *id)
 			err := resumeFn(ctx, &job)
-			if err := job.FinishedWith(ctx, err); err != nil {
+			if _, isDuplicate := errors.Cause(err).(*duplicateRegistrationError); isDuplicate {
+				// Another turn of the adoption loop already resumed this job. Swallow
+				// the error, as the job is properly resumed.
+				//
+				// This happens because job registration is asynchronous. This
+				// goroutine, not the adoption loop's goroutine, is responsible for
+				// calling Registry.register. There's a window where resumeFn has not
+				// yet registered the job with the registry, so the next turn of the
+				// adoption loop will see it holds the lease on a job that's not
+				// running, and attempt to resume it again. This likely never happens in
+				// practice because DefaultAdoptInterval is several orders of magnitude
+				// larger than the delay between resuming a job and that job registering
+				// itself. In tests, though, double resumption is a real possibility, as
+				// the DefaultAdoptInterval gets turned down an order of magnitude.
+				//
+				// TODO(benesch): make the adoption loop synchronously register the jobs
+				// it resumes. This requires API changes to "invert control"; see the
+				// TODOs in jobs.go for details.
+			} else if err := job.FinishedWith(ctx, err); err != nil {
 				// Nowhere to report this error but the log.
 				log.Errorf(ctx, "job %d: ignoring FinishedWith error: %+v", *id, err)
 			}
@@ -323,11 +342,19 @@ func (r *Registry) newLease() *Lease {
 	return &Lease{NodeID: nodeID, Epoch: r.mu.epoch}
 }
 
+type duplicateRegistrationError struct {
+	jobID int64
+}
+
+func (e *duplicateRegistrationError) Error() string {
+	return fmt.Sprintf("job %d is already registered", e.jobID)
+}
+
 func (r *Registry) register(jobID int64, j *Job) error {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 	if _, ok := r.mu.jobs[jobID]; ok {
-		return errors.Errorf("already tracking job ID %d", jobID)
+		return &duplicateRegistrationError{jobID: jobID}
 	}
 	r.mu.jobs[jobID] = j
 	return nil

--- a/pkg/sql/jobs/registry_test.go
+++ b/pkg/sql/jobs/registry_test.go
@@ -134,7 +134,7 @@ func TestRegistryRegister(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if err := registry.register(42, &Job{}); !testutils.IsError(err, "already tracking job ID") {
+	if err := registry.register(42, &Job{}); !testutils.IsError(err, "job 42 is already registered") {
 		t.Fatalf("expected 'already tracking job ID', but got '%s'", err)
 	}
 

